### PR TITLE
Fix the non-determinism in test_dwarf_expr.

### PR DESCRIPTION
**Pull request by ltfish**

### CODE REVIEW

**Comment by ltfish** [comment]
CI fails because of PR #323 is not on this branch.

File: test/test_dwarf_expr.py [hunk 1]
```diff
@@ -53,8 +53,11 @@ def test_basic_sequence(self):
         self.assertEqual(self.visitor.dump_expr([0x1d, 0x1e, 0x1d, 0x1e, 0x1d, 0x1e]),
             'DW_OP_mod; DW_OP_mul; DW_OP_mod; DW_OP_mul; DW_OP_mod; DW_OP_mul')
 
-        self.assertEqual(self.visitor.dump_expr([0x08, 0x0f, 0xe0]),
-            'DW_OP_const1u: 15; DW_OP_GNU_push_tls_address')
+        self.assertIn(self.visitor.dump_expr([0x08, 0x0f, 0xe0]),
+                      ('DW_OP_const1u: 15; DW_OP_GNU_push_tls_address',
+                       'DW_OP_const1u: 15; DW_OP_lo_user',
```
Comment (eliben) [inline test/test_dwarf_expr.py:58]: The formatting is a bit weird here. Could you just tack the closing `))` to the end of the last line?

File: test/test_dwarf_expr.py [hunk 2]
```diff
@@ -53,8 +53,11 @@ def test_basic_sequence(self):
         self.assertEqual(self.visitor.dump_expr([0x1d, 0x1e, 0x1d, 0x1e, 0x1d, 0x1e]),
             'DW_OP_mod; DW_OP_mul; DW_OP_mod; DW_OP_mul; DW_OP_mod; DW_OP_mul')
 
-        self.assertEqual(self.visitor.dump_expr([0x08, 0x0f, 0xe0]),
-            'DW_OP_const1u: 15; DW_OP_GNU_push_tls_address')
+        self.assertIn(self.visitor.dump_expr([0x08, 0x0f, 0xe0]),
```
Comment (eliben) [inline test/test_dwarf_expr.py:56]: Please add a brief comment why we're using assertIn here, and same below

**Comment by eliben** [comment]
I just went ahead and merged it because the CI is broken otherwise. I'll fix up the comments/formatting.

Thanks again!

**Pull request merged**